Classes_Homework_5/fromfile.py

```diff
@@ -0,0 +1,181 @@
+import sys
+import fileinput
+import re
+from os import remove
+
+import pyinputplus as pyip
```

Comment:
The boring recommendation about imports in PEP8 :) 

Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

Imports should be grouped in the following order:

1. Standard library imports (os/sys/re etc).
2. Related third party imports (pyinputplus).
3. Local application/library specific imports.

You should put a blank line between each group of imports.  Also, all libraries within group should **be sorted alphabetically** (asc)